src/components/Sidebar/SteemTrendingCharts.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import Trend from 'react-trend';
+import fetch from 'isomorphic-fetch';
+import _ from 'lodash';
+import Promise from 'bluebird';
+import { FormattedMessage } from 'react-intl';
+import './SteemTrendingCharts.less';
```

Comment:
Style is usually last import. Can we change that to make it concise with the rest of our app?

Reply:
sure